fission-web-server/library/Fission/Web/Server/Challenge/Verifier/Class.hs

```diff
@@ -20,15 +20,11 @@ instance MonadIO m => Verifier (Transaction m) where
       Nothing ->
         return $ Left NotFound
 
-      Just (Entity challengeId UserChallenge { userChallengeHash, userChallengeUserId }) ->
+      Just (Entity _ UserChallenge { userChallengeHash, userChallengeUserId }) ->
         if challenge == userChallengeHash
           then do
-            update userChallengeUserId
-              [ UserVerified  =. True ]
-
-            delete challengeId
```

Comment:
While we have the hood up, stop removing the challenges. It's a one-way switch (unverified -> verified), so like... are they going to verify it again or something? It's fine.